packages/next-swc/crates/emotion/src/lib.rs

```diff
@@ -337,7 +339,21 @@ impl<C: Comments> EmotionTransformer<C> {
             if index % 2 == 0 {
                 if let Some(q) = tagged_tpl.quasis.get_mut(i) {
                     let q = q.take();
-                    let minified = minify_css_string(&q.raw, index == 0, index == args_len - 1);
+                    let input = match q.cooked {
+                        Some(cooked) => cooked,
```

Comment:
This is going to lead to more headaches than expected. CSS allows `\` to escape unicode chars without a `u` prefix, eg `"\2022"` should render a `•`, and `"\0025a0"` should render a `■`. Both of these are illegal escape sequences (they're legacy octal escapes in JS), but perfectly fine in CSS.

Rather than use JS's cooking semantics, we can manually cook the raw strings using CSS semantics. We can pilfer SWC's escape parser to do this: https://github.com/swc-project/swc/blob/decebadff1686adb8ea30d0ef84d849061205c7c/crates/swc_css_parser/src/lexer/mod.rs#L892-L969